### PR TITLE
feat: add env command (#178)

### PR DESCRIPTION
**Pull request by ipapandinas**
- created a new `env` command
- moved the `check` functionality under `env` command
- implemented an `install` command to install all or individual deps
- added the *env* section to *swanky.config* on `init` with default version from docker base-image
- implemented the edit flow for the *env* section in *swanky.config*

### CODE REVIEW

File: src/commands/env/install.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      ).join(", ")}. For installing rust nightly version run: env install --deps rust@nightly`,
+      multiple: true,
+      default: [],
+      char: "d",
+    }),
+  };
+
+  constructor(argv: string[], baseConfig: any) {
+    super(argv, baseConfig);
+    (this.constructor as typeof SwankyCommand).ENSURE_SWANKY_CONFIG = false;
```
Comment (prxgr4mm3r) [inline src/commands/env/install.ts:29]: It means that the user may call this command from a non-project folder, but it throws an error that "swanky.config.json" is not found.

File: src/commands/env/install.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      );
+    }
+
+    const newDeps: Record<string, string> = {};
+    for (const item of flags.deps) {
+      const [key, value] = item.split("@");
+      newDeps[key] = value || "latest";
+    }
+
+    const localConfig = getSwankyConfig("local") as SwankyConfig;
```
Comment (prxgr4mm3r) [inline src/commands/env/install.ts:53]: There we don't check if it's project folder or not. When we are outside the project it throws an error

File: src/commands/env/install.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    const newEnv = { ...localConfig.env, ...newDeps };
```
Comment (prxgr4mm3r) [inline src/commands/env/install.ts:54]: Even if we provide exact versions and dependencies that we want by `--deps`, we anyway install all dependencies from config additionally. I think it's more logical to install only dependencies passed by `--deps` if it's used.

Comment (ipapandinas) [reply src/commands/env/install.ts]: I made the `--all` implicit (and remove it) if no dependencies are specified with the `--deps` flag. If so, only those dependencies are now installed.

**Pull request merged**